packages/cascara/src/ui/Table/0-Table.mdx

```diff
@@ -8,10 +8,9 @@ propTable: Table.js
 ## data
 
 Every object in this array will potencially be rendered as a table row.
-The shape is up to you, the only requirement is that every object (a.k.a. row) must
-have an attribute that uniquely identifies the object in the array.
+The shape is up to you, the only requirement is that every object (AKA row) must have an attribute that uniquely identifies the object in the array.
 
-> We call this attribute `uniqueIdAttribute`, which you will be able to specify later on.
+> We call this attribute **uniqueIdAttribute**, which you will be able to specify later on.
```

Comment:
When referring to a prop or component, we should show it as `code` and not bold.

Reply:
In a review call, you mentioned that it was too much highlighting of words (Since cascara's `code` format is  a little bit outstanding) so I changed it to be just **bold**.  I can change it back to `code`, np.